content/realtime/connection.textile

```diff
@@ -479,7 +479,7 @@ h6(#key).
 
 A unique private connection key @String@ used to recover or resume a connection, assigned by Ably. When recovering a connection explicitly, the <span lang="default">@recoveryKey@</span><span lang="ruby">@recovery_key@</span> is used in the @recover@ "client options":/api/realtime-sdk#client-options as it contains both the @key@ and the last message @serial@.
 
-This private connection key can also be used by other REST clients to publish on behalf of this client. See the "publishing over REST on behalf of a realtime client documentation":/rest/channels#publish-on-behalf  for more info.
+This private connection key can also be used by other REST clients to publish on behalf of this client. See the "publishing over REST on behalf of a realtime client documentation":/api/rest-sdk/channels#publish-on-behalf  for more info.
```

Comment:
`#publish-on-behalf` seems to be on [`/rest/messages`](https://ably.com/documentation/rest/messages#publish-on-behalf).